contracts/adapters/dp/MakerVault.sol

```diff
@@ -171,7 +172,7 @@ contract MakerVaultAdapter is IDebtPosition, FactoryFriendly {
      */
     uint256 public vault;
 
-    constructor(
```

Comment:
Why remove the constructor?
This means an extra call is required to initialize the contract after deployment.

Reply:
I think this comment does not apply anymore, given that we moved away from proxy deployments for adapters